public/scripts/form.js

```diff
@@ -29,14 +30,20 @@ $(function () {
 
   socket.on('logs', function (data) {
     $('#messages').append($('<li class="info">').text(data.data));
-    $("#progress").css("width", data.donePercent + "%");
+    if (data.donePercent >= 100) {
+      $("#progress").css("width", "90%");
```

Comment:
it won't go between 90 to 100. we don't know how many logs will emit in future so, I made a tap that I should not go above 90 before execution of exit code 0.